eZ/Publish/Core/Persistence/Legacy/Content/Mapper.php

```diff
@@ -363,7 +368,8 @@ private function loadCachedVersionFieldDefinitionsPerLanguage(
                 continue;
             }
 
-            $languageCodes = $this->extractLanguageCodesFromMask($languageMask, $allLanguages);
+            $allLanguagesCodes = $this->extractLanguageCodesFromMask($languageMask, $allLanguages);
+            $languageCodes = empty($translations) ? $allLanguagesCodes : array_intersect($translations, $allLanguagesCodes);
```

Comment:
I think that you need here either 
1) in L355 `array $translations = []` instead
or 
2) 
```suggestion
            $languageCodes = null !== $translations ? $allLanguagesCodes : array_intersect($translations, $allLanguagesCodes);
```

The latter one allows passing `[]` which would mean skip language codes. If this doesn't make sense in any case, than maybe we should move for 1) and leave `empty()` check here.

Reply:
I'm afraid we can't.

Ref.: https://github.com/ezsystems/ezplatform-kernel/blob/1.3/eZ/Publish/SPI/Persistence/Content/Handler.php#L63

It may be `string[]` or `null` and both, empty array and null value are treated the same way (with empty). Ref.: https://github.com/ezsystems/ezplatform-kernel/blob/1.3/eZ/Publish/Core/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php#L808